Cargo.toml

```diff
@@ -2,14 +2,14 @@
 description = "A Cassandra CQL driver, built on top of the DataStax C++ driver for performance and functionality."
 documentation = "https://docs.rs/cassandra-cpp"
 # no homepage yet
-repository = "https://github.com/Metaswitch/cassandra-rs"
+repository = "https://github.com/cassandra-rs/cassandra-rs"
 readme = "README.md"
 keywords = [ "Cassandra", "binding", "CQL", "client", "database" ]
 categories = [ "api-bindings", "database", "external-ffi-bindings", "asynchronous" ]
 license = "Apache-2.0"
 name = "cassandra-cpp"
-version = "3.0.2-pre"
-authors = ["Keith Wansbrough <kwansbrough@microsoft.com>"]
+version = "3.0.2"
```

Comment:
Should we also bump the dependency on cassandra-cpp-sys?

Reply:
Good shout - might as well.